cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -23,15 +23,14 @@ import (
 	"net/http/httptest"
 	"os"
 	"path/filepath"
-	"reflect"
 	"testing"
 	"time"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
Could you merge the three import blocks into one?

Reply:
the fmt here should be std lib, newline, our stuff, newline, external deps

Reply:
uhm... maybe I got confused with the lines that the diff adds. This is correct :)